apps/web/util/globals.ts

```diff
@@ -14,6 +14,17 @@ export const PROVIDERS = [
 		name: "Backblaze",
 		logo: "/backblaze.png",
 	},
+	{
+		id: "wasabi",
+		name: "Wasabi",
+		logo: "/wasabi.png",
+	},
+	{
+		id: "digitalocean",
+		name: "Digital Ocean",
+		logo: "/digitalocean.png",
+		isComingSoon: false,
```

Comment:
when do we need to add `isComingSoon` to an object? some objects have it when it has a value of `true`. does `isComingSoon` already default to `false`?

Reply:
isComingSoon is needed when the service provider is visioned to be implemented in the future but is not currently. It does default to false.

Reply:
I am wondering why we added `isComingSoon: false` here then if the other objects that default to false don't have it. the objects that do have `isComingSoon` here are `true`.

Reply:
isComingSoon is set to true if the provider is not yet integrated to Firefiles

Reply:
but digital ocean has been integrated, so we don't need `isComingSoon` at all, right?

Reply:
yeah, that makes sense, we can remove `isComingSoon` but however to avoid any null issues, I just have that key set to false